Tribler/community/multichain/StatisticsDisplay/dbDriver.py

```diff
@@ -0,0 +1,191 @@
+import sqlite3
```

Comment:
Add module description at the top of module. Look at [PEP 257](https://www.python.org/dev/peps/pep-0257/) for more information.